pkg/klusterlet/clusterinfo/clusterinfo_controller.go

```diff
@@ -409,36 +409,79 @@ func (r *ClusterInfoReconciler) getOCPDistributionInfo() (clusterv1beta1.OCPDist
 
 	ocpDistributionInfo.DesiredVersion, _, err = unstructured.NestedString(obj.Object, "status", "desired", "version")
 	if err != nil {
-		klog.Errorf("failed to get OCP cluster version in latest history of status: %v", err)
+		klog.Errorf("failed to get OCP desired version in clusterVersion: %v", err)
 	}
 	availableUpdates, _, err := unstructured.NestedSlice(obj.Object, "status", "availableUpdates")
 	if err != nil {
-		klog.Errorf("failed to get OCP cluster version in latest history of status: %v", err)
+		klog.Errorf("failed to get OCP availableUpdates in clusterVersion: %v", err)
 	}
 	for _, update := range availableUpdates {
-		availableVersion, _, err := unstructured.NestedString(update.(map[string]interface{}), "version")
+		versionUpdate := clusterv1beta1.OCPVersionRelease{}
+		versionUpdate.Version, _, err = unstructured.NestedString(update.(map[string]interface{}), "version")
 		if err != nil {
-			klog.Errorf("failed to get OCP cluster version in latest history of status: %v", err)
-			continue
+			klog.Errorf("failed to get version in availableUpdates of clusterVersion: %v", err)
+		}
+		versionUpdate.Image, _, err = unstructured.NestedString(update.(map[string]interface{}), "image")
```

Comment:
do we import openshift/api?  if yes, we should render to typed api instead  of using unstructured. Using nested could be erroneous.

Reply:
no, I tried to import openshift/api, but conflicted with the old version imported in hive.